nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -21,6 +21,20 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `hardware.nvidia` has a new option `open` that can be used to opt in the opensource version of NVIDIA kernel driver. Note that the driver's support for GeForce and Workstation GPUs is still alpha quality, see [NVIDIA Releases Open-Source GPU Kernel Modules](https://developer.nvidia.com/blog/nvidia-releases-open-source-gpu-kernel-modules/) for the official announcement.
 
+- As Singularity has renamed to Apptainer to distinguish from an un-renamed fork,
+  there is now three packages of Singularity/Apptainer:
+  * `singularity-legacy`: From `hpcng/singularity`, which is the original repo and is now redirected to `apptainer/singularity`. The import path is `github.com/hpcng/singularity`.
+  * `apptainer`: From `apptainer/apptainer`, which is the new repo after renaming. The import path is `github.com/apptainer/apptainer`
+  * `singularity-ce`: From `sylabs/singularity`, which is the fork by Sylabs Inc.. The import path is `github.com/sylabs/singularity`
```

Comment:
Do we really need all of them or could we for example drop the legacy one without problems?

Reply:
The github description of `apptainer/singularity` says:

> Singularity has been EOL'ed, see Apptainer

Thus there are only two "variants" that are not officially EOL: `apptainer` from apptainer, and `singularity` from sylabs.
Let's just keep those